### PR TITLE
Update repository texts

### PR DESCRIPTION
**Pull request by kain88-de**
This is a larger change of the whole text that we have available here. The changes are based on my experiences this year, where I noticed we can streamline things more or where information was missing / split between documents. 

Big changes inlcude

- No need to have a draft on this github repo. Students may do this at their own leisure
- More detailed information for sub-orgs, who can apply, what information we would like, mentoring
- removed as much information for specific GSoC years as possible (simpler update for next year)
- added more information about expectations towards students during the application phase.

### CODE REVIEW

**Comment by kain88-de** [comment]
ping @rgaiacs

File: templates/ideas-page.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Project Ideas
+
+###  Idea Title 
+
+#### Abstract
+
+One line project.
+
+| **Intensity**                          | **Priority                | **Involves**  | **Mentors**              |
```
Comment (ferrine) [inline templates/ideas-page.md]: is it a typo here? `**Priority`

Comment (kain88-de) [inline templates/ideas-page.md]: yes thats a typo thanks

File: organization/operations.md [hunk 2]
```diff
@@ -11,81 +11,56 @@ if approved by NumFOCUS board.
...
+### How to sign up mentors
+
+You can use google forms to sign up mentors who want to work with NumFOCUS. This
+will have the advantage that we automatically have a list of all mentors and
+their contact information. Mentors should be given access to numfocus mentoring
+mailing list.
+
+## Timeline for Organization Administrators
+
+*January*: Ask NumFOCUS if they want to participate. Preferably with help of NumFOCUS staff
```
Comment (MridulS) [inline organization/operations.md]: @kain88-de 
Once we have the go ahead from NumFOCUS we need to have a decent amount of project ideas to make our GSoC application strong. I suggest we ask sub orgs (admins) who have regularly participated in GSoC to help us with the ideas page.

- Start preparing an ideas page with the sub orgs admins.

**Pull request merged**